src/app_engine/app.yaml

```diff
@@ -9,6 +9,14 @@ handlers:
   static_files: html/\1
   upload: html/(.*\.html)
 
+- url: /(.*\.json)
+  static_files: html/\1
+  upload: html/(.*\.json)
```

Comment:
The key files should not be in the html/ folder. But I agree we should be
more specific just to be safe.

On Thu, Feb 12, 2015 at 9:20 AM, Chuck Hays <notifications@github.com>
wrote:

> In src/app_engine/app.yaml
> <https://github.com/webrtc/apprtc/pull/66#discussion_r24598888>:
>
> > @@ -9,6 +9,14 @@ handlers:
> >    static_files: html/\1
> >    upload: html/(.*\.html)
> >
> > +- url: /(.*\.json)
> > +  static_files: html/\1
> > +  upload: html/(.*\.json)
>
> This makes me nervous because we are storing things like the gcm server
> key in json files. Could we add a specific file name here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/apprtc/pull/66/files#r24598888>.
>



-- 
Jiayang Liu